vllm/entrypoints/openai/serving_chat.py

```diff
@@ -304,6 +307,7 @@ async def create_chat_completion(
                         lora_request=lora_request,
                         trace_headers=trace_headers,
                         priority=request.priority,
+                        data_parallel_rank=data_parallel_rank,
```

Comment:
Update the signature below? https://github.com/vllm-project/vllm/blob/2e5d21378db8226b04bb76f3e7119a9ddd0239df/vllm/engine/protocol.py#L51-L59